WatchDogIntelliJPlugin/WatchDog/resources/META-INF/plugin.xml

```diff
@@ -1,7 +1,7 @@
 <idea-plugin version="2">
     <id>nl.tudelft.watchdog</id>
     <name>WatchDog</name>
-    <version>1.5.0</version>
+    <version>1.5.0.1</version>
```

Comment:
Let's bump it up to 1.6.0 immediately, and we'll include the fix that we create on Tuesday with it.